README.md

```diff
@@ -39,6 +39,23 @@ The properties are also available in templates:
 {{/if}}
 ```
 
+### is-mobile Helper
+
+The is-mobile helper can be used as an alternative to `isMobile` Service.
+It expects one argument to be passed as a param.
+
+```handlebars
+{{#if (is-mobile 'any')}}
+  I'm on a mobile device
+{{/if}}
+```
+
+```handlebars
+<div class="{{if (is-mobile 'android' 'is-android')}} {{if (is-mobile 'apple' 'is-apple')}}">
```

Comment:
I forgot to add this. Are the parentheses here correct? Shouldn't it be like this?
```hbs
<div class="{{if (is-mobile 'android') 'is-android'}} {{if (is-mobile 'apple') 'is-apple'}}">
```

Reply:
Done 👍